### PR TITLE
Updated html css feature

### PR DESCRIPTION
**Pull request by Bigizi**
Portfolio Project.
This project has the following files: an index HTML file, a style CSS file, a READMe file, and other configuration files.
I made changes to the index file where I added more sections to complete the task and I styled them in the style CSS file.
N.B: there is no other user on my account. crispin12 and Bigizi both are my names I don't know what problem happens to my git cause I have two emails one bigizinduwayo@gmail.com(this is one is registered for microverse platforms) and bigizinduwa@gmail.com. It seems my GitHub is linked to both emails, so I am trying to solve this issue but I failed to do. so understand that I am the owner of my GitHub account.

### CODE REVIEW

**Review by amejid — COMMENTED** [review]
Hi @Bigizi, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has commits from another Github user.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by huseyinbiyik** [comment]
Hi @Bigizi 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Alt Text

## Invalid Code Review Request

Please use the[ Microverse Readme Template](https://github.com/microverseinc/readme-template) and replace all the placeholders, add the MIT license file.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by huseyinbiyik** [comment]
Hi @Bigizi 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Alt Text

## Invalid Code Review Request

- Kindly use [Microverse Readme template](https://github.com/microverseinc/readme-template) and make sure all the sections exist in yours and placeholders are replaced with your project and personal data.  This is crucial to have a professional and instructive repository. 👌

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by ha-manel — COMMENTED** [review]
# STATUS: INVALID 🔴 
Hi @Bigizi 
I'm sure you put a lot of effort into this project but this PR is invalid because your README file is not following the template😓kindly use the following [template](https://raw.githubusercontent.com/microverseinc/readme-template/master/README.md) (just copy this and paste it in your README file, it will be formatted accordingly) then fill out all the sections with information relevant to your project 👍   

Once you're ready, please submit the PR for review again 👍 
Best of luck!

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1,24 +1,129 @@
...
+Clone this repository to your desired folder:
+
+### Install
+
+Install this project with:
+
+clone the repo using this command:
+
+git clone
+
```
Comment (christianbiring1) [inline README.md:77]: - [ ] Kindly consider walking the reader of your README through the process they need to go through to use your repo locally.
`Hint: To clone the repos git clone <repos_link>`
`To navigate cd <directory_name>`
`...`

File: README.md [hunk 2]
```diff
@@ -1,24 +1,129 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/Bigizi/Portfolio)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/bigizi-nduwayo-crispin-74b534227/)
```
Comment (christianbiring1) [inline README.md:94]: - [ ] Kindly make sure you give the correct link to your social handles
`hint:[name](://github/<your_username>)`

File: README.md [hunk 3]
```diff
@@ -1,24 +1,129 @@
...
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+
+# 📖 [Crispin_Portfolio] <a name="about-project"></a>
+
+**[Crispin_Portfolio]** is a project requested to every student at Microverse for the purpose of building knowledge capacity and for the job searching or job market
```
Comment (christianbiring1) [inline README.md:27]: - [ ] Kindly make sure the description is telling what the project is about.
You may consider referencing to the description of the `setup_mobile first project`

File: index.html [hunk 4]
```diff
@@ -20,16 +20,16 @@
         <section class="first-section">
             <h1 id="my-h1">I’m Crispin<br/> Glad to see<br/> you!</h1>
```
Comment (christianbiring1) [inline index.html:21]: - [ ] Kindly make sure you stick to the original design as much as possible. your typos(font), color, and spacing must be similar to the original design.
`Hint: To see the style on a specific element in Figma, just click on that element and on the right side you will have the styles applied to it.`
Figma design :point_down: 
image

Your design :point_down: 
image

File: index.html [hunk 5]
```diff
@@ -39,29 +39,234 @@ <h1 id="my-h1">I’m Crispin<br/> Glad to see<br/> you!</h1>
...
+            </div>
+            <div>
+                <ul class="list2-container">
+                    <li class="lists">html</li>
+                    <li class="lists">css</li>
+                    <li class="lists">javascript</li>
+                </ul>
+            </div><br/>
+            <div>
+                <button id="my-btn">See Project</button>
```
Comment (christianbiring1) [inline index.html:92]: - [ ] You may also consider fixing the border-radius of your buttons

YOur design :point_down: 
image

Figma template :point_down: 
image

File: index.html [hunk 6]
```diff
@@ -39,29 +39,234 @@ <h1 id="my-h1">I’m Crispin<br/> Glad to see<br/> you!</h1>
...
                     </a>
                 </li>
             </ul>
+        </section><br/><br/><br/>
+        <section class="second-section">
+            <img src="images/Snapshoot Portfolio.png" alt="screenshot"/>
+            <ul class="section2-list">
+                <div>
+                    <h2 class="my-h2">Tonic</h2>
+                </div>
```
Comment (christianbiring1) [inline index.html:70]: - [ ] For semantic reasons, Kindly consider removing this `div` in the list element `ul`, only `li` is supposed to be directly in a `ul`.
You may consider placing `ul` after this `div`
`EVERYWHERE YOU DID THAT, KINDLY CORRECT IT.`

File: index.html [hunk 7]
```diff
@@ -39,29 +39,234 @@ <h1 id="my-h1">I’m Crispin<br/> Glad to see<br/> you!</h1>
...
+                    </ul>
+                </div>
+                <div>
+                    <ul>
+                        <li class="about">
+                            <p class="paragraph">Frameworks</p>
+                            <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+                                <path fill-rule="evenodd" clip-rule="evenodd" d="M5.29289 8.29289C5.68342 7.90237 6.31658 7.90237 6.70711 8.29289L12 13.5858L17.2929 8.29289C17.6834 7.90237 18.3166 7.90237 18.7071 8.29289C19.0976 8.68342 19.0976 9.31658 18.7071 9.70711L12.7071 15.7071C12.3166 16.0976 11.6834 16.0976 11.2929 15.7071L5.29289 9.70711C4.90237 9.31658 4.90237 8.68342 5.29289 8.29289Z" fill="#5E6C84"/>
+                            </svg>
+                        </li><hr/>
```
Comment (christianbiring1) [inline index.html:257]: - [ ] Kindly consider readjusting the color of this line.

Your design :point_down: 
image

Figma design :point_down: 
image

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @Bigizi , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
soclose


## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `summary` :+1: 
- [ ] Good commit message

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Giving your PR a descriptive title telling what the project aims to do.
`Hint: Finish mobile version would sound better than update HTML CSS feature`

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**